en/flight_controller/beaglebone_blue.md

```diff
@@ -65,102 +58,88 @@ Currently running on a:
 MODEL_BB_BLUE
 Robot Control library Version:
 1.0.0
-
 ```
 
-As of this writing, the latest OS images in which librobotcontrol works properly
-is [bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz](https://rcn-ee.net/rootfs/bb.org/testing/2018-07-22/stretch-iot/bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz).
+> **Tip** Optionally you can update to a realtime kernel, and if you do, re-check if *librobotcontrol* works properly with the realtime kernel.
 
-#### Setup Robot Control Library
+The latest OS images in which *librobotcontrol* works properly (at time of writing) is [bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz](https://rcn-ee.net/rootfs/bb.org/testing/2018-07-22/stretch-iot/bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz).
 
-If you want to build px4, there are additional setup steps for this library.
 
-It seems that currently librobotcontrol debian package is only available on 
-BeagleBoard products. Here are the ways to obtain the librobotcontrol on 
-BeagleBone Blue:
+### Setup Robot Control Library
 
-1. use the one pre-installed in BeagleBoard images
+If you want to build PX4, there are additional setup steps for this library.
 
-2. install from debian package or repository 
-```sh
-    sudo apt update && sudo apt install librobotcontrol
-```
+At time of writing the *librobotcontrol* debian package is only available on BeagleBoard products (not BeagleBone Blue). 
+Here are the ways to obtain the *librobotcontrol* on BeagleBone Blue:
 
-3. install from source
-```sh
-    git clone https://github.com/StrawsonDesign/librobotcontrol.git
-    cd librobotcontrol
-    sudo make install
-```
+1. Use the one pre-installed in BeagleBoard images.
+1. Install from debian package or repository:
+   ```sh
+   sudo apt update && sudo apt install librobotcontrol
+   ```
+1. Install from source
+   ```sh
+   git clone https://github.com/StrawsonDesign/librobotcontrol.git
+   cd librobotcontrol
+   sudo make install
+   ```
 
 After acquiring the pre-built library,
 
-* select librobotcontrol installation directory, and define it as LIBROBOTCONTROL_INSTALL_DIR environment variable so that other unwanted headers will not be included
-
-* install robotcontrol.h and rc/* into $LIBROBOTCONTROL_INSTALL_DIR/include
-
- install pre-built native (ARM) version of librobotcontrol.* into $LIBROBOTCONTROL_INSTALL_DIR/lib
+1. Select the *librobotcontrol* installation directory, and set it in the `LIBROBOTCONTROL_INSTALL_DIR` environment variable so that other unwanted headers will not be included
```

Comment:
@UAV-Pilot Is this just on the BBB for native builds or something we have to do on dev computer too?

My guess is that you probably need to do it on the computer you are building on.

Reply:
If it's cross compile then these steps only need be done on cross compile dev computer, otherwise only on native board.  And yes, user need do it on the machine that compiles PX4.